### DISCUSSION TITLE
Sintering neck calculation

### DISCUSSION THREAD

**Discussion by HPL001** [idaholab/moose#29555] [Q&A Modules: Phase field]
Dear moose community,
hello. I am doing the sintering calculation between two particles, I want to output the change curve of the sintering neck in the post-processing, how should I operate?do I need to use NodalVariableValue or something?

**Comment by GiudGiud** [comment]
hello

In the postprocessing? So you want to modify the mesh then or just compute a new value for an existing quantity?
`NodalVariableValue` lets you output a value. If you want to use this value into a new postprocessor, you can use the ParsedPostprocessor
